src/hb-ot-var-fvar-table.hh

```diff
@@ -280,6 +280,32 @@ struct fvar
     return axisCount;
   }
 
+  void collect_name_ids (hb_set_t *nameids) const
+  {
+    if (!has_data ()) return;
+
+    + get_axes ()
```

Comment:
Here I believe you should be able to do
| hb_map (&AxisRecord::axisNameId)
| hb_sink (nameids)
;